CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.53.0] - 2021-03-12
+### Added
+- gRPC: accept keepalive parameters for gRPC outbound configuration.
+- observability: deprecation of logging levels `applicationError` and `serverError` in favor
```

Comment:
We are missing change-log for caller-procedure changes made by @gopalchouhan, could you please update change-log?

Reply:
True good call, let me add something